daemon/volumes_unix.go

```diff
@@ -270,109 +214,19 @@ func (daemon *Daemon) verifyVolumesInfo(container *Container) error {
 	return nil
 }
 
-// parseVolumesFrom ensure that the supplied volumes-from is valid.
-func parseVolumesFrom(spec string) (string, string, error) {
-	if len(spec) == 0 {
-		return "", "", derr.ErrorCodeVolumeFromBlank.WithArgs(spec)
+// setBindModeIfNull is platform specific processing to ensure the
+// shared mode is set to 'z' if it is null. This is called in the case
+// of processing a named volume and not a typical bind.
+func setBindModeIfNull(bind *volume.MountPoint) *volume.MountPoint {
```

Comment:
ewwwwwwwww

what if we extract that `"z"` to a constant per platform and set it with something like this:

```go
if bind.Mode == "" {
   bind.Mode = defaultBindMode
}
```